src/main/groovy/com/vherasymenko/avro/decoder/core/AvroDecoderService.groovy

```diff
@@ -0,0 +1,25 @@
+package com.vherasymenko.avro.decoder.core
+
+import event.course_install.CourseInstall
+import org.apache.avro.generic.GenericDatumReader
+import org.apache.avro.generic.GenericRecord
+import org.apache.avro.io.DatumReader
+import org.apache.avro.io.DecoderFactory
+
+/**
+ * The avro decoder.
+ */
+class AvroDecoderService implements AvroDecoderPort {
+
+    @Override
+    void decodeEvent( String event ) {
+        def inputStream = new ByteArrayInputStream( event.bytes )
```

Comment:
Actually this line is wrong. I haven't noticed before that there is the method that takes the string as the input:
```
JsonDecoder jsonDecoder(Schema schema, String input)
```
I'll change this in the next PR.